components/Layout.tsx

```diff
@@ -375,8 +347,13 @@ const Layout: FC<
         </NavBar>
       )}
 
-      {/* Log out Dialog */}
+      {/* Dialogs */}
       <LogOutDialog open={logOutOpen} onClose={() => setLogOutOpen(false)} />
+      <ReportIssueDialog
+        open={reportOpen}
+        onClose={() => setReportOpen(false)}
+        onSubmit={() => setReportOpen(false)}
+      />

```

Comment:
Changing to using the Report Issue Dialog Component is good for making the report form easier to access and also making the code easier to make changes to. 